content/en/user-guide/tools/transparent-endpoint-injection/dns-server.md

```diff
@@ -82,60 +120,114 @@ Disabling SSL validation may have undesired side effects and security implicatio
 Make sure to use this only for local testing, and never in production.
 {{< /alert >}}
 
+
+## Customizing internal endpoint resolution
+
+The DNS name `localhost.localstack.cloud` (and any subdomains like `mybucket.s3.localhost.localstack.cloud`) is used internally in LocalStack to route requests, e.g., between a Lambda container and the LocalStack APIs.
+
+Please refer to the steps in the [Route53 docs]({{< ref "route53" >}}) for more details on how the internal DNS name can be customized.
```

Comment:
I'm not sure I follow what this section is documenting. It is talking about how we use `localhost.localstack.cloud` internally to communicate between LocalStack components, but then it mentions how this value can be customised by Route53. I would suggest that mentioning Route53 is _additive_, but also that `LOCALSTACK_HOST` can be used to customise the actual domain used.

Reply:
@dfangl knows all the legacy history once he is back ;)

Reply:
FYI: I ended up re-phrasing it in the new [LocalStack endpoints section](https://docs.localstack.cloud/user-guide/tools/dns-server/#localstack-endpoints) based on @simonrw suggestion of mentioning `LOCALSTACK_HOST`.

@dfangl feel free to chime in if you remember what the real purpose was 🪄 :)